cogs/utils/embed_handler.py

```diff
@@ -0,0 +1,25 @@
+from discord import Color, Embed, Member, Status
+
+from bot.cogs.utils.members import get_member_activity, get_member_roles_as_mentions, get_member_status
+
+
+def status_embed(member: Member, *, description: str = "") -> Embed:
+    """
+    Construct status embed for certain member.
+    """
```

Comment:
Single line docstrings should only take a single line
```suggestion
    """Construct status embed for a certain member. """
```